ui.apps/src/main/content/jcr_root/apps/core/cif/components/content/teaser/v1/teaser/clientlib/editor/js/teaser.js

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ *
+ *    Copyright 2020 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+($ => {
+    'use strict';
+
+    const dialogContentSelector = '[data-cmp-is="commerceteaser-editor"].cmp-teaser__editor';
+    const productFieldSelector = '[data-cmp-teaser-v1-dialog-edit-hook="actionLink"][placeholder="Product"]';
+    const categoryFieldSelector = '[data-cmp-teaser-v1-dialog-edit-hook="actionLink"][placeholder="Category"]';
+    const actionsMultifieldSelector = '.cmp-teaser__editor-multifield_actions';
+    const actionsMultifieldItemSelector = '.coral3-Multifield-item';
```

Comment:
Please check GRANITE-20375. You're not supposed to use any selectors with `.coral-*`, `.coral3-*` and `.granite-*`.